### DISCUSSION TITLE
feat: `ipfs refs` command not supported?

### DISCUSSION THREAD

**Discussion by koxon** [ipfs/helia#499] [Q&A] [answered]
I don't see the equivalent of the `ipfs refs` command in Helia.

See the official doc:
https://docs.ipfs.tech/reference/kubo/cli/#ipfs-refs

`ipfs-js` had it implemented. Am I missing something or it's not implemented yet in Helia?
Any plans on supporting it?

**Comment by 2color** [comment, marked as answer]
Helia does not come with a built in RPC (or REST) API like js-ipfs.

Depending on the codec of your data, you can use other codec-specific methods to find references to other CIDs. 

With UnixFS, you can use the [`ls` method](https://helia.io/interfaces/_helia_unixfs.index.UnixFS.html#ls) to iterate of UnixFS nodes.

Can you share more information about what you're trying to do?
